docs/content/packages/styled-system.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Styled System
+status: Experimental 
+source: 'https://github.com/primer/octicons/tree/v2/lib/octicons_styled'
+---
+
+[![npm version](https://img.shields.io/npm/v/@primer/styled-octicons/canary.svg)](https://www.npmjs.org/package/@primer/styled-octicons)
+
+The `@primer/styled-octicons` package wraps icon components from [`@primer/octicons-v2-react`](/packages/react) with [system props](https://primer.style/components/system-props), making them easier to style in projects that use [styled system](https://styled-system.com/)—like [Primer Components](https://primer.style/components).
+
+## Install
+
+```shell
+npm install @primer/styled-octicons@canary
```

Comment:
Why do we recommend folks to install the canary?

Reply:
Until the [`v2` branch](https://github.com/primer/octicons/pull/370) is merged, there won't actually be a `@latest` version of the package. When the `v2` branch is merged, we'll update the docs to remove `@canary`.

Reply:
🙌